packages/browser/go/jsutil/jsutil.go

```diff
@@ -47,16 +46,22 @@ func WrapInPromise(f func() (interface{}, error)) js.Value {
 // InefficientlyConvertToJS converts the given Go value to a JS value by
 // encoding to JSON and then decoding it. This function is not very efficient
 // and its use should be phased out over time as much as possible.
-func InefficientlyConvertToJS(value interface{}) (js.Value, error) {
-	var jsValue interface{}
-	buf := bytes.Buffer{}
-	if err := json.NewEncoder(&buf).Encode(value); err != nil {
-		return js.Undefined(), err
-	}
-	if err := json.NewDecoder(&buf).Decode(&jsValue); err != nil {
+func InefficientlyConvertToJS(value interface{}) (result js.Value, err error) {
```

Comment:
Do you have a sense of how inefficient this is compared to other `JSValue` functions? How deserving is this function of its name?

Reply:
No, as I mentioned in the PR description, we should explore that separately.